get-involved.php

```diff
@@ -144,7 +144,7 @@
     </div>
 </section>
 
-<section id="desktop-development" class="grey">
```

Comment:
I don't think we should change the IDs here. This is not user-facing and this change makes this ID more ambiguous than it was before

Reply:
Indeed. I just wanted to keep them in sync with the actual visible labels, as those small details can add brittleness over time. But yeah, it *is* a small detail, so no objection to revert this.